packages/cli/src/Logger.ts

```diff
@@ -1,31 +1,48 @@
-import * as winston from 'winston';
 import config = require('../config');
+import * as winston from 'winston';
 
 import {
 	ILogger,
 	LogTypes,
 } from 'n8n-workflow';
 
-export class Logger implements ILogger {
+class Logger implements ILogger {
 	private logger: winston.Logger;
-	
+
 	constructor() {
+		const level = config.get('logs.level');
+		const output = (config.get('logs.output') as string).split(',').map(output => output.trim());
 
-		const logFormat = winston.format.printf(({ message }) => {
-			return message;
-		}) as winston.Logform.Format;
-		
 		this.logger = winston.createLogger({
-			level: config.get('logs.level'),
-			format: logFormat,
-			transports: [
-				new winston.transports.Console(),
-			],
+			level,
 		});
-		
-		if (config.get('logs.output') === 'file') {
+
+		if (output.includes('console')) {
+			let format: winston.Logform.Format;
+			if (['debug', 'verbose'].includes(level)) {
+				format = winston.format.combine(
+					winston.format.metadata(),
+					winston.format.timestamp(),
+					winston.format.colorize({ all: true }),
+					winston.format.printf(({ level, message, timestamp, metadata }) => {
+						return `${timestamp} | ${level.padEnd(18)} | ${message}` + (Object.keys(metadata).length ? ` ${JSON.stringify(metadata)}` : '');
```

Comment:
nitpick - spacing here seems off

Reply:
Hi @mutdmour 

I did not find any issue here, spacing seems fine (once before and after each pipe). Any suggestions?

Reply:
my bad. probably just looked funky on github